README.md

```diff
@@ -437,7 +437,7 @@ $ npm i -D @types/github-script@github:actions/github-script
 And then add the `jsDoc` declaration to your script like this:
 ```js
 // @ts-check
-/** @param {import('@types/github-script').AsyncFunctionArguments} AsyncFunctionArguments */
+/** @param {import('github-script').AsyncFunctionArguments} AsyncFunctionArguments */
```

Comment:
Does this not need to match the installed dependency above?

```shell
npm i -D @types/github-script@github:actions/github-script
```

Reply:
The `@types` packages are automatically mapped to the packages with `@types` stripped off.

And importing `@types/github-script` causes error TS6137. Rewriting to `github-script` resolves this issue.

ref: https://github.com/microsoft/TypeScript/issues/16472

Reply:
How you doing?